crates/hotshot/examples/web-server-da/types.rs

```diff
@@ -22,43 +20,22 @@ use std::fmt::Debug;
 #[derive(Clone, Debug, Deserialize, Serialize, Hash, PartialEq, Eq)]
 pub struct NodeImpl {}
 
-pub type ThisLeaf = Leaf<DemoTypes>;
-pub type ThisMembership =
-    GeneralStaticCommittee<DemoTypes, ThisLeaf, <DemoTypes as NodeType>::SignatureKey>;
+pub type ThisMembership = GeneralStaticCommittee<DemoTypes, <DemoTypes as NodeType>::SignatureKey>;
```

Comment:
Nit: `libp2p/types.rs` uses the same type alias, so I'm wondering if we can put it somewhere else to avoid duplicate definitions. Maybe in the same file as `DemoTypes` and rename it to `DemoMembership`?